integration/all/top_blockio_test.go

```diff
@@ -79,12 +72,30 @@ func TestTopBlockIO(t *testing.T) {
 	t.Parallel()
 	ns := GenerateTestNamespaceName("test-top-block-io")
 
-	// TODO: Handle it once we support getting container image name from docker
-	isDockerRuntime := IsDockerRuntime(t)
+	var extraArgs string
+	expectedEntry := &topblockioTypes.Stats{
+		Comm:  "dd",
+		Write: true,
+	}
+
+	switch DefaultTestComponent {
+	case IgTestComponent:
+		extraArgs = "--runtimes=" + containerRuntime + " -m 999"
```

Comment:
Mmmm [the issue](https://github.com/inspektor-gadget/inspektor-gadget/pull/1408) with `interval == timeout` wasn't related with `max-rows`.

I just remember why we were using `max-rows` in only `ig`. It's because we are not filtering by namespace so other tests running in parallel could interfere:

https://github.com/inspektor-gadget/inspektor-gadget/blob/009a293fb23dbb19af859017e418da2fe4b17fe9/integration/ig/k8s/top_file_test.go#L97-L101

So, @pawarpranav83, please keep adding `m` only for `ig` and add the previous TODO comment. Thanks!